data/mods/Tanks/items.json

```diff
@@ -1,27 +1,41 @@
 [
   {
+    "abstract": "artillery_casing",
+    "type": "GENERIC",
+    "category": "spare_parts",
+    "name": "A large artillery casing.",
+    "stackable": true,
+    "material": "steel",
```

Comment:
Artillery shells can be made of brass, steel, aluminum, and a wide variety of alloys.